2.Lets-Build-OTPublisher/Lets-Build-OTPublisher/ViewController.m

```diff
@@ -74,7 +74,7 @@ - (BOOL)shouldAutorotateToInterfaceOrientation:
  */
 - (void)doConnect
 {
-    OTError *error;
+    OTError *error = nil;
```

Comment:
I want to know that too. I never get this problem since Apple implemented native nilling for object pointers. Probably this could be CLANG option or something like that